router/router_test.go

```diff
@@ -36,9 +36,16 @@ func ensureHasKey(t *testing.T, data map[string]json.RawMessage, key string) {
 	}
 }
 
+type mockGetAliasBidderNames struct{}
+
+func (m mockGetAliasBidderNames) getAliasBidderNames() map[openrtb_ext.BidderName]openrtb_ext.BidderName {
+	return map[openrtb_ext.BidderName]openrtb_ext.BidderName{openrtb_ext.BidderName("alias"): openrtb_ext.BidderName("parentAlias")}
+}
+
 func TestNewJsonDirectoryServer(t *testing.T) {
-	alias := map[string]string{"aliastest": "appnexus"}
-	handler := NewJsonDirectoryServer("../static/bidder-params", &testValidator{}, alias)
+	getAliasBidders = mockGetAliasBidderNames{}
+	defaultAlias := map[string]string{"aliastest": "appnexus"}
```

Comment:
should not limit `getAliasBidderNames()` to return same result.

```
type mockGetAliasBidderNames struct {
   aliasToParentMap map[openrtb_ext.BidderName]openrtb_ext.BidderName   
}

func (m *mockGetAliasBidderNames) getAliasBidderNames() 
	return  m.aliasToParentMap
}

func TestNewJsonDirectoryServer(t *testing.T) {
   getAliasBidders := &mockGetAliasBidderNames{
         openrtb_ext.BidderName("alias"): openrtb_ext.BidderName("parentAlias"),
    }
   ....
 
```

Reply:
Not needed anymore. Removed in https://github.com/prebid/prebid-server/pull/3055/commits/d5c207f18d210f765ed3301ddcc857da973a8b60